### DISCUSSION TITLE
Change all textStyles

### DISCUSSION THREAD

**Discussion by ghost** [Sketchware-Pro/Sketchware-Pro#265] [Ideas]
All ```android:textStyle="bold"``` to ```style="@style/TextAppearance.AppCompat.Title"```

**Comment by khaled-0** [comment]
in all decompiled files? sure, a project wide find replace would do.
but what about non decompiled files

**Comment by j-px** [comment]
okay, but why lol

**Reply by ghost** [reply]
text style bold, too bold

**Comment by DerGoogler** [comment]
Why not opening an issue?

**Comment by Ninjacoderhsi** [comment]
کس ننت

**Comment by Ninjacoderhsi** [comment]
> okay, but why lol

کیرم ب ناموست خوب؟
